typescript/submissions.d.ts

```diff
@@ -210,3 +211,15 @@ export type FormStoreDefinition = {
   formId: number
   formElements: FormElementWithName[]
 }
+
+export type NewFormSubmissionPayment = {
+  submissionId: string
+  formId: number
+  type: PaymentSubmissionEvent['type']
+  status: 'PENDING' | 'SUCCEEDED' | 'FAILED'
+}
+export type FormSubmissionPayment = NewFormSubmissionPayment & {
+  createdAt: string
+  updatedAt: string
+  paymentTransaction: unknown
```

Comment:
we coming back to type this later?

Reply:
Will it have a specific type? I assume it would change per payment provider

Reply:
It would be nice to type it for SDK purposes. I say we replace the `type` property with this `paymentTransaction` property and type it per `PaymentSubmissionEvent` type.